### ISSUE TITLE
Missing underline on service dashboard for failed messages

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#828]
## Missing underline on service dashboard for failed messages 

In admin, For "1712 text messages sent" you can hover / click anywhere in the blue, and the text is underlined, but for "427 failed" you have to click on the text, and nothing is underlined

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

SEV-2  

## To Reproduce

1. Go to a service dashboard
2. Try to hover over the failed email or text line
3. See that no underline appears

## Impact

Users might have a harder time finding out why messages failed because they don't know they can select for more information. They miss the failure page entirely.

This would also impact accessibility because without the underline, users who use assistive tech might not know there is a link there.

If applicable

Impact on Notify users: Users might have a harder time finding out why messages failed because they don't know they can select for more information. They miss the failure page entirely.

Impact on Recipients: None

Impact on Notify team: None

## Screenshots

![image (7).png](https://images.zenhubusercontent.com/618bd78afce6ad0aa17d429c/fc3ebe23-3dae-471c-bf22-02ce3a26b9f6)

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/828?planning-poker) with Zenhub @amazingphilippe @jzbahrai @sastels
